### ISSUE TITLE
Hazelcast and Redisson cannot be upgraded to latest opentracing contrib versions

### ISSUE THREAD

**Issue by malafeev** [opentracing-contrib/java-specialagent#191]
For example upgrade opentracing-hazelcast to 0.1.5:
```
<dependency>
      <!-- https://github.com/opentracing-contrib/java-hazelcast -->
      <groupId>io.opentracing.contrib</groupId>
      <artifactId>opentracing-hazelcast</artifactId>
      <version>0.1.5</version>
</dependency>
```

Then run unit tests (`HazelcastTest`).
Error:
```
java.lang.NoClassDefFoundError: com/hazelcast/cp/CPSubsystem
at
io.opentracing.contrib.specialagent.rule.hazelcast.HazelcastAgentIntercept.getOneInstance(HazelcastAgentIntercept.java:27)
```

**Comment by SevaSafris** [comment]
@malafeev:
> "latest hazelcast and redisson requires new rules like for lettuce"

**Comment by malafeev** [comment]
fixed via #202
